gray-theme/unauthenticated/gray-theme.css

```diff
@@ -422,6 +422,10 @@ details.ui_hidden_table_start > summary::-webkit-details-marker {
     color: #a1acc0;
 }
 
+details:not([open]).on-hover:not(:hover) summary::after {
```

Comment:
Do we still need this if the description column is either always shown or always hidden?